crypto/asn1/a_int.c

```diff
@@ -289,7 +289,7 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
             ASN1err(ASN1_F_ASN1_GET_INT64, ASN1_R_TOO_SMALL);
             return 0;
         }
-        *pr = -(int64_t)r;
+        *pr = -(uint64_t)r;
```

Comment:
This is bound to be flagged as warning by some of the compilers we "care" about. Should be `0 - (uint64_t)r`.